### PR TITLE
feat(api): Add accountStatus check in graphql api

### PR DESCRIPTION
**Pull request by vbudhram**
## Because

- We want to expose this functionality in graphql

## This pull request

- Adds the eqalivent `GET /account/status` endpoint in graphql

## Issue that this pull request solves

Closes: https://mozilla-hub.atlassian.net/browse/FXA-6332

## Checklist

- [x] My commit is GPG signed.
- [x] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

File: packages/fxa-auth-server/test/mail_helper.js [hunk 1]
```diff
@@ -66,8 +66,9 @@ module.exports = (printLogs) => {
             console.log(`Notification email: ${template}`);
           } else {
             console.error('\x1B[31mNo verify code match\x1B[39m');
+            console.error(mail);
```
Comment (vbudhram) [inline packages/fxa-auth-server/test/mail_helper.js:69]: Reverting a change I accidently added in a previous PR.

File: packages/fxa-graphql-api/src/gql/account.resolver.ts [hunk 2]
```diff
@@ -509,6 +516,34 @@ export class AccountResolver {
...
+    description:
+      'Check the status of an account using session token or uid. ' +
+      'This query is equivalent to the GET /account/status endpoint in auth-server.',
+  })
+  @CatchGatewayError
+  public async accountStatus(
+    @Args('input', { type: () => AccountStatusInput })
+    input: AccountStatusInput
+  ): Promise<AccountStatusPayload> {
+    if (input.token) {
```
Comment (vbudhram) [inline packages/fxa-graphql-api/src/gql/account.resolver.ts:529]: In auth-server the `/account/status` route is considered optional, therefore I couldn't use the `@GqlSessionToken` decorator. This route can accept a sessionToken id or a uid and check to see if that account exists.

File: packages/fxa-graphql-api/src/gql/dto/payload/account-status.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Field, ObjectType } from '@nestjs/graphql';
+
+@ObjectType()
+export class AccountStatusPayload {
+  @Field({
+    description: 'A unique identifier for the client performing the mutation.',
+    nullable: true,
+  })
+  public clientMutationId?: string;
```
Comment (chenba) [inline packages/fxa-graphql-api/src/gql/dto/payload/account-status.ts:12]: accountStatus is not a mutation so I don't believe this applies.

File: packages/fxa-graphql-api/src/gql/dto/input/account-status.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class AccountStatusInput {
+  @Field({
+    description: 'A unique identifier for the client performing the mutation.',
+    nullable: true,
+  })
+  public clientMutationId?: string;
```
Comment (chenba) [inline packages/fxa-graphql-api/src/gql/dto/input/account-status.ts:12]: I don't think we need this since it's not a mutation.

**Review by chenba — APPROVED** [review]
LGTM.  r+wc

**Pull request merged**